terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/pagination/PaginationInfoTest.java

```diff
@@ -110,11 +111,10 @@ public void testCreateAttributeMap_SortSet() {
 
         int page = 1;
         int size = 1;
-        Sort mockedSort = new Sort(Direction.DESC, "id");
 
         // run
         Map<String, Object> attributesMap = PaginationInfo.createAttributeMap(
-                page, size, mockedSort);
+                page, size, by(Direction.DESC, "id"));
```

Comment:
- [ ] You should not use `import static org.springframework.data.domain.Sort.by`.
Because this class is not understood to use implicitly.